plugins/woocommerce/includes/admin/class-wc-admin-profile.php

```diff
@@ -222,6 +222,16 @@ public function save_customer_meta_fields( $user_id ) {
 						update_user_meta( $user_id, $key, wc_clean( $_POST[ $key ] ) );
 					}
 				}
+
+				/**
+				 * Hook: woocommerce_customer_save_address.
+				 *
+				 * Fires when an address is edited via a customer's user profile admin screen.
+				 *
+				 * @param int    $user_id User ID being saved.
+				 * @param string $load_address Type of address e.g. billing or shipping.
+				 */
+				do_action( 'woocommerce_customer_save_address', $user_id, $load_address );
```

Comment:
Some things we need to consider here:

- Seems like `$load_address` is not defined so, as currently written, this would lead to an error. 
- Also, with an eye on PHP 8.x compat, we would ideally want it to be named in a way that matches other instances of the hook (so, `$address_type` is probably a better name).
- Currently, in the outermost `foreach()` loop, we only fetch the value. However, if we also fetch the key, we can use that to determine the address type. In other words, we could do something like `foreach ( $save_fields as $fieldset_type => $fieldset )`. Typically, `$fieldset_type` will be either `billing` or `shipping` and we can then map it to `$address_type` a little later on (but see also the next point).
- Last but not least, the result of a call to `WC_Admin_Profile::get_customer_meta_fields()` is filterable, and so we may get back additional fields besides the `shipping` and `billing` address fields. Given the hook name, we probably don't want to fire it unless we're looking at one of those default fieldsets (hypothetical example: if a third party plugin adds support for customer meta data like `previous_purchases`, we would not want to fire the save address hook).